src/ebay-fake-menu/menu.tsx

```diff
@@ -1,16 +1,12 @@
-import React, {
-    Children, cloneElement,
-    ComponentProps, FC, ReactElement, KeyboardEvent, MouseEvent
-} from 'react'
+import React, { Children, cloneElement, ComponentProps, FC, ReactElement, KeyboardEvent } from 'react'
 import classNames from 'classnames'
 import { EbayFakeMenuItemProps } from './index'
 
 type SpanProps = Omit<ComponentProps<'div'>, 'onKeyDown'>
-type Callback = (event: KeyboardEvent | MouseEvent, i: number) => void
 type Props = SpanProps & {
     itemMatchesUrl?: boolean;
-    onKeyDown?: Callback;
-    onSelect?: Callback;
+    onKeyDown?: (i: number, event: KeyboardEvent) => void
+    onSelect?: (i: number) => void;
```

Comment:
See my previous comment to keep the event as the first param for onSelect and onKeyDown. Besides, when you change event handler signature, it will introduce breaking changes on ebay-fake-menu.